apps/avatax/src/lib/error-utils.ts

```diff
@@ -1,13 +1,53 @@
 import { TRPCClientError } from "@trpc/client";
+import { GraphQLError } from "graphql";
+import { BaseError } from "../error";
+import { logger } from "../logger";
+import { CalculateTaxesPayload } from "../modules/webhooks/payloads/calculate-taxes-payload";
+import { OrderCancelledPayload } from "../modules/webhooks/payloads/order-cancelled-payload";
+import { OrderConfirmedPayload } from "../modules/webhooks/payloads/order-confirmed-payload";
 
-function resolveTrpcClientError(error: unknown) {
+export function resolveTrpcClientError(error: unknown) {
   if (error instanceof TRPCClientError) {
     return error.message;
   }
 
   return "Unknown error";
 }
 
-export const errorUtils = {
-  resolveTrpcClientError,
-};
+export class SubscriptionPayloadErrorChecker {
+  static SubscriptionPayloadError = BaseError.subclass("SubscriptionPayloadError");
+
+  constructor(
+    private injectedLogger: Pick<typeof logger, "error">,
+    private injectedErrorCapture: (expection: any) => void,
```

Comment:
- "exception" typo
- error is typed, so probably you should replace "any" with instance of this error